Puppetfile

```diff
@@ -2,14 +2,14 @@ forge 'https://forgeapi.puppetlabs.com'
 
 # Dependencies
 mod 'puppetlabs/mysql',         :git => 'https://github.com/theforeman/puppetlabs-mysql', :ref => '3.11.x'
```

Comment:
Sadly this is still needed. I've added it to the discussion in https://etherpad.openstack.org/p/puppet-community-triage and I hope that it works.